pocs/camera/camera.py

```diff
@@ -80,7 +102,7 @@ def process_exposure(self, *args, **kwargs):
         raise NotImplementedError
 
     def __str__(self):
-        return "{}({}) on {}".format(self.name, self.uid, self.port)
+        return "{} ({}) on {}".format(self.name, self.uid, self.port)
```

Comment:
Just the name of the focuser is fine, I just think that when printing the representation of a camera it should at least be obvious that it does have a focuser attached.